packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -161,7 +168,9 @@ const SimpleStatistic: FC<SimpleStatisticsProps> = ({
                             marginTop: 10,
                             display: 'flex',
                             alignItems: 'center',
-                            color: comparisonValueColor,
+                            color: `${
+                                flipColors ? comparisonValueColor : 'inherit'
```

Comment:
should we check the flip logic in `comparisonValueColor` ? 
```
            case ComparisonDiffTypes.POSITIVE:
                return flipColors ? Colors.RED3 : Colors.GREEN3;
            case ComparisonDiffTypes.NEGATIVE:
                return flipColors ? Colors.GREEN3 : Colors.RED3;
```

Reply:
I completely misunderstood the requirements 🥲 will re-work this

Reply:
eg: in this case the comparison value should be red

![Captura de ecrã 2023-05-12, às 10 39 58](https://github.com/lightdash/lightdash/assets/9117144/9a22941c-0055-4062-83ec-87d3a6f4b679)

Reply:
got it!